### PR TITLE
Make all the basic classes and config

### PR DESCRIPTION
**Pull request by reyostallenberg**

### CODE REVIEW

**Comment by reyostallenberg** [comment]
This is just the base, need to clean up and implement much more

File: src/Controller/RegistrationController.php [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Component\Security\Core\Encoder\UserPasswordEncoderInterface;
+
+class RegistrationController extends AbstractController
+{
+    /**
+     * @Route("/register", name="app_register")
```
Comment (gisostallenberg) [inline src/Controller/RegistrationController.php]: I think the name should relate to the bundle
```suggestion
     * @Route("/register", name="connectholland_userbundle_register")
```

File: src/Controller/RegistrationController.php [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+     */
+    public function register(Request $request, UserPasswordEncoderInterface $passwordEncoder): Response
+    {
+        $user = new User();
```
Comment (gisostallenberg) [inline src/Controller/RegistrationController.php]: Can this (or will this) be a service?

File: composer.json [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+        "symfony/form": "^3.4 || ^4.3",
+        "symfony/framework-bundle": "^3.4 || ^4.3",
+        "symfony/http-kernel": "^3.4 || ^4.3",
+        "symfony/security-bundle": "^3.4 || ^4.3",
+        "symfony/templating": "^3.4 || ^4.3",
+        "symfony/translation": "^3.4 || ^4.3",
+        "symfony/twig-bundle": "^3.4 || ^4.3",
+        "symfony/validator": "^3.4 || ^4.3",
+        "twig/twig": "^2.0 || ^3.0"
+    },
```
Comment (gisostallenberg) [inline composer.json]: Maybe you should analyse this using https://packagist.org/packages/maglnet/composer-require-checker, I see getDoctrine in the code, but I think it is missing here.

File: src/Controller/SecurityController.php [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Component\Security\Http\Authentication\AuthenticationUtils;
+
+class SecurityController extends AbstractController
+{
+    /**
+     * @Route("/login", name="app_login")
```
Comment (gisostallenberg) [inline src/Controller/SecurityController.php]: See above

File: composer.json (hunk 3 above)
Comment (reyostallenberg) [reply composer.json]: Thanks, will do it for a following PR

File: src/Controller/RegistrationController.php (hunk 2 above)
Comment (reyostallenberg) [reply src/Controller/RegistrationController.php]: Maybe, need to look in to all implementation

**Pull request merged**